### ISSUE TITLE
Multi-period tests for `process_sync_committee_updates`

### ISSUE THREAD

**Issue by michaelsproul** [ethereum/eth2.0-specs#2330]
Presently there's 1 test for `process_sync_committee_updates` that checks the transition from period 0 to period 1. This is an exceptional case, because the `base_epoch` for both period 0 and period 1 is epoch 0.

In Lighthouse, I had written an erroneous bounds check that looked like this:

```rust
 if base_epoch != current_base_epoch
     && base_epoch != current_base_epoch.safe_add(spec.epochs_per_sync_committee_period)?
 {
    return Err(Error::EpochOutOfBounds);
 }
 ```
 
It failed as soon as I tried transitioning from period 1 to period 2, because `process_sync_committee_updates` tried to calculate the sync committee for period 3, which has `base_epoch == 2 * EPOCHS_PER_SYNC_COMMITTEE_PERIOD` which is not equal to the `current_base_epoch` (0) + `EPOCHS_PER_SYNC_COMMITTEE_PERIOD`.

I think it would be good to test a period transition from a period other than 0 to catch mistakes like this.

**Comment by djrtwo** [comment]
added a couple of cases in #2334

**Comment by michaelsproul** [comment]
Another issue I just noticed: the sync aggregate epoch processings tests lack a `meta.yaml`, yet they _need_ BLS enabled because constructing a sync committee involves real signature aggregation.

**Comment by djrtwo** [comment]
closed via #2334
